lib/serum/build/file_copier.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Serum.Build.FileCopier do
+  @moduledoc false
```

Comment:
given that this is a public interface, maybe documenting it would be a good idea

Reply:
I don't put `@moduledoc false` with no reason. That module is for internal uses only.

Reply:
But yes, After I have decided which modules to expose and which modules to hide, I added an underscored variable (`_moduledocp`) to internal modules like this (because adding a module attribute `@moduledocp` and not using it results in a compiler warning):

https://github.com/Dalgona/Serum/blob/15ec9b55bd2c26096ea44a89154379f167ba7d8b/lib/serum/dev_server/service.ex#L4-L6

So it will be later added to this module too.